project/dexter-vd-cpp/src/java/com/samsung/sec/dexter/vdcpp/checkerlogic/MagicNumberDataType64BitCheckerLogic.java

```diff
@@ -0,0 +1,131 @@
+ *   this list of conditions and the following disclaimer in the documentation
+ *   and/or other materials provided with the distribution.
+ * 
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+package com.samsung.sec.dexter.vdcpp.checkerlogic;
+
+import org.apache.log4j.Logger;
+import org.eclipse.cdt.core.dom.ast.ASTVisitor;
+import org.eclipse.cdt.core.dom.ast.IASTBinaryExpression;
+import org.eclipse.cdt.core.dom.ast.IASTDeclaration;
+import org.eclipse.cdt.core.dom.ast.IASTDeclarator;
+import org.eclipse.cdt.core.dom.ast.IASTExpression;
+import org.eclipse.cdt.core.dom.ast.IASTFunctionDefinition;
+import org.eclipse.cdt.core.dom.ast.IASTIdExpression;
+import org.eclipse.cdt.core.dom.ast.IASTLiteralExpression;
+import org.eclipse.cdt.core.dom.ast.IASTName;
+import org.eclipse.cdt.core.dom.ast.IASTNode;
+import org.eclipse.cdt.core.dom.ast.IASTTranslationUnit;
+import org.eclipse.cdt.core.dom.ast.IBinding;
+import org.eclipse.cdt.core.dom.ast.IVariable;
+import org.eclipse.cdt.core.dom.ast.IType;
+import org.eclipse.cdt.core.dom.ast.IBasicType;
+import org.eclipse.cdt.core.dom.ast.IValue;
+import org.eclipse.cdt.core.dom.ast.IASTEqualsInitializer;
+import org.eclipse.cdt.core.dom.ast.IASTStatement;
+import org.eclipse.cdt.core.dom.ast.IASTReturnStatement;
+import org.eclipse.cdt.core.dom.ast.IASTDeclSpecifier;
+import org.eclipse.cdt.core.dom.ast.IASTSimpleDeclSpecifier;
+import org.eclipse.cdt.core.dom.ast.IASTNamedTypeSpecifier;
+
+import com.samsung.sec.dexter.core.analyzer.AnalysisConfig;
+import com.samsung.sec.dexter.core.analyzer.AnalysisResult;
+import com.samsung.sec.dexter.core.checker.Checker;
+import com.samsung.sec.dexter.vdcpp.plugin.DexterVdCppPlugin;
+import com.samsung.sec.dexter.vdcpp.util.CDTASTUtil;
+
+
+public class MagicNumberDataType64BitCheckerLogic implements ICheckerLogic{
+
+	private IASTTranslationUnit translationUnit;
+	private final static Logger LOG = Logger.getLogger(DexterVdCppPlugin.class);
+
+	@Override
+	public void analyze(final AnalysisConfig config, final AnalysisResult result, 
+			final Checker checker, final IASTTranslationUnit unit) {
+		translationUnit =unit;
+		
+		ASTVisitor visitor = createVisitor(config, result, checker);
```

Comment:
This is recommendation.

You can set 'shouldVisitDeclarations' as true, in createVisitor method body.
If you use more specific name of method instead of createVisitor, that's better.
For example, 'createReturnStatementOnFunctionVisitor'. Is it too long?  :-)
Or you can express the purpose of the method.